frontend/src/metabase/status/components/StatusListing/StatusListing.tsx

```diff
@@ -1,13 +1,23 @@
 import React from "react";
 import DatabaseStatus from "../../containers/DatabaseStatus";
+import FileUploadStatus from "../FileUploadStatus";
 import { StatusListingRoot } from "./StatusListing.styled";
 
 export interface StatusListingProps {
   isAdmin: boolean;
 }
 
 const StatusListing = ({ isAdmin }: StatusListingProps): JSX.Element => {
-  return <StatusListingRoot>{isAdmin && <DatabaseStatus />}</StatusListingRoot>;
+  return (
+    <StatusListingRoot>
+      {isAdmin && (
```

Comment:
I think we're planning to allow uploads for anyone with write permissions on the uploads db, right? I guess this requires the settings PR to be merged to be able to check.